### ISSUE TITLE
Error "Illegal instruction (4)" after logging in

### ISSUE THREAD

**Issue by atroxix** [nextcloud/server#11165]
### Steps to reproduce
1. Open web browser without an active Nextcloud session
2. Log in

### Expected behaviour
Nextcloud should function properly.

### Actual behaviour
Getting "internal error", Apache logs the following error on each access:
```[Tue Sep 11 10:58:02.006363 2018] [core:notice] [pid 37197] AH00052: child pid 37372 exit signal Illegal instruction (4)```

### Server configuration

**Operating system**: FreeBSD 11.2

**Web server:** Apache 2.4

**Database:** MariaDB 10.3.8

**PHP version:** 7.2.9

**Nextcloud version:** 14.0.0

**Updated from an older Nextcloud/ownCloud or fresh install:** Upgraded from 13.0.6

**Where did you install Nextcloud from:** FreeBSD ports

```
# sudo -u www php occ integrity:check-core
# sudo -u www php occ app:list | while read line; do app="`echo "$line" | awk '{print $2}' | awk -F ':' '{print $1}'`" ; if [ "$app" = "" ]; then continue ; else echo "Checking '$app'" ; fi ; sudo -u www php occ integrity:check-app $app ; done                                               
Checking 'accessibility'
Checking 'activity'
Checking 'cloud_federation_api'
Checking 'comments'
Checking 'dav'
Checking 'federatedfilesharing'
Checking 'files'
Checking 'files_pdfviewer'
Checking 'files_sharing'
Checking 'files_texteditor'
Checking 'files_trashbin'
Checking 'files_versions'
Checking 'files_videoplayer'
Checking 'gallery'
Checking 'lookup_server_connector'
Checking 'notifications'
Checking 'oauth2'
Checking 'provisioning_api'
Checking 'sharebymail'
Checking 'support'
Checking 'theming'
Checking 'twofactor_backupcodes'
Checking 'updatenotification'
Checking 'workflowengine'
Checking 'admin_audit'
Checking 'encryption'
Checking 'federation'
Checking 'files_external'
Checking 'firstrunwizard'
Checking 'logreader'
Checking 'nextcloud_announcements'
Checking 'password_policy'
Checking 'serverinfo'
Checking 'socialsharing_email'
  - EXCEPTION:
    - class: OC\IntegrityCheck\Exceptions\InvalidSignatureException
    - message: Signature data not found.
Checking 'socialsharing_facebook'
  - EXCEPTION:
    - class: OC\IntegrityCheck\Exceptions\InvalidSignatureException
    - message: Signature data not found.
Checking 'survey_client'
Checking 'systemtags'
Checking 'user_external'
Checking 'user_ldap'
```
</details>

**List of activated apps:**
<details>

```
# sudo -u www php occ app:list
Enabled:
  - accessibility: 1.0.1
  - activity: 2.7.0
  - cloud_federation_api: 0.0.1
  - comments: 1.4.0
  - dav: 1.6.0
  - federatedfilesharing: 1.4.0
  - files: 1.9.0
  - files_pdfviewer: 1.3.2
  - files_sharing: 1.6.2
  - files_texteditor: 2.6.0
  - files_trashbin: 1.4.1
  - files_versions: 1.7.1
  - files_videoplayer: 1.3.0
  - gallery: 18.1.0
  - lookup_server_connector: 1.2.0
  - notifications: 2.2.1
  - oauth2: 1.2.1
  - provisioning_api: 1.4.0
  - sharebymail: 1.4.0
  - support: 1.0.0
  - theming: 1.5.0
  - twofactor_backupcodes: 1.3.1
  - updatenotification: 1.4.1
  - workflowengine: 1.4.0
Disabled:
  - admin_audit
  - encryption
  - federation
  - files_external
  - firstrunwizard
  - logreader
  - nextcloud_announcements
  - password_policy
  - serverinfo
  - socialsharing_email
  - socialsharing_facebook
  - survey_client
  - systemtags
  - user_external
  - user_ldap</summary>
```
</details>

**Nextcloud configuration:**
<details>
<summary>Config report</summary>

```
# sudo -u www php occ config:list system
{
    "system": {
        "instanceid": "***REMOVED SENSITIVE VALUE***",
        "passwordsalt": "***REMOVED SENSITIVE VALUE***",
        "datadirectory": "***REMOVED SENSITIVE VALUE***",
        "dbtype": "mysql",
        "version": "14.0.0.19",
        "dbname": "***REMOVED SENSITIVE VALUE***",
        "dbhost": "***REMOVED SENSITIVE VALUE***",
        "dbtableprefix": "oc_",
        "dbuser": "***REMOVED SENSITIVE VALUE***",
        "dbpassword": "***REMOVED SENSITIVE VALUE***",
        "installed": true,
        "theme": "",
        "trusted_domains": [
            "...",
            "...",
            "..."
        ],
        "forcessl": false,
        "mail_smtpmode": "smtp",
        "updatechecker": false,
        "secret": "***REMOVED SENSITIVE VALUE***",
        "singleuser": false,
        "loglevel": 2,
        "logfile": "\/var\/log\/nextcloud.log",
        "trashbin_retention_obligation": "auto",
        "filelocking.enabled": "true",
        "memcache.locking": "\\OC\\Memcache\\APCu",
        "memcache.local": "\\OC\\Memcache\\APCu",
        "mail_from_address": "***REMOVED SENSITIVE VALUE***",
        "mail_domain": "***REMOVED SENSITIVE VALUE***",
        "overwrite.cli.url": "https:\/\/...\/",
        "htaccess.RewriteBase": "\/new",
        "mysql.utf8mb4": true,
        "maintenance": false,
        "apps_paths": [
            {
                "path": "\/usr\/local\/www\/nextcloud\/apps",
                "url": "\/apps",
                "writable": true
            },
            {
                "path": "\/usr\/local\/www\/nextcloud\/apps-pkg",
                "url": "\/apps-pkg",
                "writable": false
            }
        ],
        "mail_smtpauthtype": "LOGIN",
        "mail_smtphost": "***REMOVED SENSITIVE VALUE***"
    }
}
```
</details>

**Are you using external storage, if yes which one:** no

**Are you using encryption:** no

**Are you using an external user-backend, if yes which one:** no

</details>

### Client configuration
**Browser:** Chrome, Opera, Nextcloud client

**Operating system:** Windows 7

#### Web server error log
<details>
<summary>Web server error log</summary>

```
[Tue Sep 11 12:01:29.917852 2018] [core:notice] [pid 40728] AH00052: child pid 41152 exit signal Illegal instruction (4)
```
</details>

#### Nextcloud log (data/nextcloud.log)
<details>
<summary>Nextcloud log</summary>
The last entry:
```
{"reqId":"DNQM54coJbKNgHkeYTxv","level":1,"time":"2018-09-11T07:37:04+00:00","remoteAddr":"10.0.254.1","user":"--","app":"updater","method":"GET","url":"\/core\/ajax\/update.php?requesttoken=...","message":"\\OC\\Updater::resetLogLevel: Reset log level to Warning(2)","userAgent":"Mozilla\/5.0 (Windows NT 6.1; Win64; x64) AppleWebKit\/537.36 (KHTML, like Gecko) Chrome\/68.0.3440.106 Safari\/537.36 OPR\/55.0.2994.56","version":"14.0.0.19"}
```
</details>

**Comment by timm2k** [comment]
You have to provide the app-id you want to check...

```sudo -u www php occ integrity:check-app calendar```

**Comment by atroxix** [comment]
Okay, I updated my original information in that place.

**Comment by atroxix** [comment]
One more thing about it...

At first I could successfully login, and as apps socialsharing_email and socialsharing_facebook were disabled I went on to enable them.
When I clicked on Update button, I was asked password, and as my password is quite long, I got an error message stating that typing password timed out (or something like that).
I did it a few times and then suddenly got that "Illegal instruction" error.

Does this perhaps help in guessing what could have gone wrong?
